bigquery/bigquery_access_quickstart/grant_access_table_or_view.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bigqueryaccessquickstart
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/bigquery"
+	"cloud.google.com/go/iam"
+)
+
+func grantAccessToResource(w io.Writer, projectID, datasetID, resourceID string) error {
+
+	// [START bigquery_grant_access_to_table_or_view]
```

Comment:
issue: move region tag START to before import statement.

See: https://googlecloudplatform.github.io/samples-style-guide/#imports